src/proto/CMakeLists.txt

```
@@ -7,6 +7,12 @@ if (${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
     list(APPEND CMAKE_PREFIX_PATH "${Protobuf_PREFIX_PATH}")
 endif()
 find_package(Protobuf REQUIRED)
+if ("${Protobuf_VERSION}" MATCHES [[[0-9]+.([0-9]+).[0-9]+]])
+    if(CMAKE_MATCH_1 GREATER_EQUAL 22)
```

Is it possible to use built-in `VERSION_LESS`?

```suggestion
if (NOT "${Protobuf_VERSION}" VERSION_LESS 22)
```

---

@bukepo  Thank you for your suggestion. Use built-in "VERSION_LESS" to check the version of protobuf. Please see the updated change.